linux_os/guide/system/software/integrity/crypto/harden_sshd_crypto_policy/rule.yml

```diff
@@ -0,0 +1,20 @@
+documentation_complete: true
+
+title: 'Harden SSHD Crypto Policy'
+
+description: |-
+    Crypto Policies are means of enforcing certain cryptographic settings for selected applications including OpenSSH server.
+    The SSHD service is by default configured to enhance its configuration based on currently configured Crypto-Policy. However, in certain cases it is required to override the Crypto Policy specific to OpenSSH Server.
+    This can be done by dropping a file named <tt>opensshserver-xxx.config</tt>, replacing <tt>xxx</tt> with arbitrary identifier, into <tt>/etc/crypto-policies/local.d</tt>. This has to be followed by running <tt>update-crypto-policies</tt> sothat changes are applied.
+    Changes are propagated into <tt>/etc/crypto-policies/back-ends/opensshserver.config</tt>. This rule checks if this file contains predefined <tt>CRYPTO_POLICY</tt> environment variable configured with predefined value.
+
+rationale: |-
+    The Common Criteria requirements specify that certain parameters for OpenSSH Server are configured e.g. supported ciphers, accepted host key algorithms, public key types, key exchange algorithms, HMACs and GSSAPI key exchange is disabled.
```

Comment:
I think the rationale for this rule is that algorithms required by Common Criteria are stricter than what is set in any current crypto policy.